2021-10-13-github-templates.md

```diff
@@ -0,0 +1,123 @@
+# OmbuLabs Standard Github Templates
+
+## Problem:
+
+New and junior contributors might be unsure about the information we require to
+efficiently review a PR. Similarly, it might be unclear what information is required when
+a new BUG or FEATURE is logged as an issue in Github.
+
+We have existing PR and ISSUE templates but they are inconsistent and might ask for
+unnecessary information from the contributor.
+
+The goal is to simplify  and streamline the steps required to make a contribution or log
+an issue.
+
+## Proposal:
+
+Come up with a consistent set of Github templates. The templates should include prompts
+only the ESSENTIAL information we require. We want to make sure that we don't ask for more
+information than we require - as this will discourages contributions.
+
+## Templates:
+
+### Pull Requests
+
+```
+    <!--- Before we start... -->
+    <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+    <!--- Did you update any relevant documentation -->
+    <!--- Did you add tests to cover new changes -->
+
+    ## Description
+    <!--- Describe your changes in detail -->
+    <!--- Please read the README before submitting pull requests for this project. -->
+
+    ## Motivation and Context
+    <!--- Why is this change required? What problem does it solve? -->
+    <!--- If it fixes an open issue, please link to the issue here. -->
+
+    ## How Has This Been Tested?
+    <!--- Include any relevant details about your testing environment and the steps you followed -->
+
+    ## Screenshots (if appropriate):
+
+    **I will abide by the [code of conduct](<LINK TO CODE OF CONDUCT HERE>)**
+```
+
+
+### Issues: Bug Reports
+
+```
+    <!--- Before we start... -->
+    <!--- Did you check the documentation for answers? -->
+    <!--- Did you make sure that this bug has not already been reported? -->
+
+    ## Expected Behavior
+    <!--- Tell us what should happen -->
+
+    ## Actual Behavior
+    <!--- Tell us what happens instead -->
+
+    ## Possible Fix
+    <!--- Optionally suggest a fix or reason for the bug -->
+
+    ## Steps to Reproduce
+    <!--- Provide a link to a live example, or an unambiguous set of steps to -->
```

Comment:
This sounds to me like a way to get people to not submit issues because they don't check all the boxes... So maybe just rephrase it like this: 

```
<!--- Do you have a live example in a sample repo? If so, please provide a link to it here. Otherwise, please provide a set of steps to -->
```